src/Symfony/Contracts/Translation/Test/TranslatorTest.php

```diff
@@ -59,6 +59,8 @@ public function testTransChoiceWithExplicitLocale($expected, $id, $number)
     }
 
     /**
+     * @requires extension intl
```

Comment:
why ? the polyfill should be enough as they work in the en locale.

Reply:
that's the contracts, we don't want to add the polyfill to it.

Reply:
then maybe change that to a check on the class existence (`@requires class NumberFormatter`) so that it accepts both the polyfill or the extension ?

Reply:
We already have this `@requires` in the very next test case.
And we don't care to make this testable with the polyfill - that wouldn't provide anything more AFAIK.